static/js/components/EmploymentForm.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import Dialog from 'material-ui/Dialog';
+import Card from 'react-mdl/lib/Card/Card';
+import Switch from 'react-mdl/lib/Switch';
+import FABButton from 'react-mdl/lib/FABButton';
+import Icon from 'react-mdl/lib/Icon';
+import IconButton from 'react-mdl/lib/IconButton';
+import _ from 'lodash';
+import moment from 'moment';
+
+import { saveAndContinue } from '../util/profile_edit';
+import { generateNewWorkHistory } from '../util/util';
+import EmploymentValidation from  '../util/EmploymentValidation';
+import { ProfileFormFields } from '../util/ProfileFormFields';
+
+class EmploymentForm extends ProfileFormFields(EmploymentValidation) {
```

Comment:
It feels like the inheritance structure in these classes is getting complicated. Is there a way we can move the validation pieces into a standalone class which can be used by the other classes? We should generally favor composition over inheritance because the relationships are made more explicit.